src/messaging/tests/TestReliableMessageProtocol.cpp

```diff
@@ -462,8 +462,6 @@ void CheckUnencryptedMessageReceiveFailure(nlTestSuite * inSuite, void * inConte
 
     // Since peer dropped the message, we might have pending acks. Let's clear the table
     rm->ClearRetransTable(rc);
```

Comment:
This line would need to go away too: "rc" is a deleted object here, because the exchange is closed.

To keep this test working as designed, we should probably keep the manual close and pass the "expect response" flag to `SendMessage` to keep the exchange alive after the send.